crates/tabby-scheduler/src/lib.rs

```diff
@@ -8,17 +8,19 @@ mod indexer;
 use async_stream::stream;
 pub use code::CodeIndexer;
 use crawl::crawl_pipeline;
-use doc::SourceDocument;
+use doc::create_web_index;
+pub use doc::{DocIndexer, WebDocument};
 use futures::StreamExt;
-use indexer::{IndexAttributeBuilder, Indexer};
+pub use indexer::{IndexAttributeBuilder, Indexer};
```

Comment:
```suggestion
use indexer::{IndexAttributeBuilder, Indexer};
```

Please always pay more attention to visibility modifiers - it's critical to decoupling